tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -73,6 +73,24 @@ def test_simple(self):
         assert response.status_code == 200, response.content
         assert [attrs for time, attrs in response.data["data"]] == [[{"count": 1}], [{"count": 2}]]
 
+    def test_misaligned_last_bucket(self):
+        response = self.client.get(
+            self.url,
+            data={
+                "start": iso_format(self.day_ago - timedelta(minutes=30)),
+                "end": iso_format(self.day_ago + timedelta(hours=1, minutes=30)),
+                "interval": "1h",
+            },
+            format="json",
+        )
+
+        assert response.status_code == 200, response.content
+        assert [attrs for time, attrs in response.data["data"]] == [
```

Comment:
I know we don't have this pattern in the other tests, but should we check the timestamps in the response too for this test? (or atleast their deltas)

Reply:
I don't think that is necessary. For this specific test, the exact timestamps aren't important. The key here is that the last data point exists to encapsulate all the data.